File: dsa_lab_1/class2/main2.c
#include <stdio.h>

int main() {
    //data array
    int DATA[] = {10, 20, 30, 40, 50};
    int ITEM;

    int N = sizeof(DATA) / sizeof(DATA[0]);
    int LOC = 0;
    

    printf("Enter the item to search for: ");
    scanf("%d", &ITEM);


    for (int i = 0; i < N; i++) {
        if (DATA[i] == ITEM) {
            LOC = i + 1;
            break;
        }
    }
    
    if (LOC == 0) {
        printf("Item is not in the array data\n");
    } else {
        printf("LOC is the location of item: %d \n", LOC);
    }
    
    return 0
}

File: dsa_lab_1/class2/main2.py
DATA = [10,12,20,50,30,60]
N = 5
LOC=0
k=1
item=10
flag=0

for i in range(5):
	# print(i)
	if DATA[i]==item:
		flag=1
if flag==1:
	print("found")
else:
	print("not found")